hack/dind

```diff
@@ -26,8 +26,33 @@ if ! mountpoint -q /tmp; then
 fi
 
 if [ $# -gt 0 ]; then
-	exec "$@"
+	cat > /lib/systemd/system/hack-dind.target << EOF
+[Unit]
+Description=the target for hack-dind-entrypoint.service
+EOF
+	cat > /lib/systemd/system/hack-dind-entrypoint.service << EOF
+[Unit]
+Description=hack-dind-entrypoint.service ($@)
+After=hack-dind.target
+
+[Service]
+Type=oneshot
```

Comment:
This seems making `systemctl status` to be "initializing". Not sure how to make the status "running".

Reply:
Removing Type=oneshot solved the issue